spyder/config/main.py

```diff
@@ -282,7 +282,8 @@
               'search_text': [''],
               'search_text_samples': [TASKS_PATTERN],
               'more_options': True,
-              'case_sensitive': False
+              'case_sensitive': False,
+              'search_in_index': 0

```

Comment:
There's no need to add this here because a) there's no need to define a different value for this in the future; and b) you're already giving it a default value in the plugin.